ci-operator/step-registry/cucushift/installer/check/ibmcloud/cucushift-installer-check-ibmcloud-commands.sh

```diff
@@ -29,7 +29,15 @@ if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+set +e
+HasError=0
 
 checkProviderType
 
-exit $?
\ No newline at end of file
+((HasError+=$?))
```

Comment:
Why we need this change?

Reply:
> Why we need this change?

I want to use 'HasError' to record all the test results in check_ibmcloud.

Reply:
But currently there is only 1 check in this step.